tests/src/test/scala/whisk/core/invoker/ActionLogDriverTests.scala

```diff
@@ -77,8 +77,8 @@ class ActionLogDriverTests
             raw"""|{"time":"","stream":"stdout","log":"a"}
                   |{"time":"","stream":"stdout","log":"b"}
                   |{"time":"","stream":"stdout","log":"c"}
-                  |{"time":"","stream":"stdout","log":"$LOG_ACTIVATION_SENTINEL\n"}
-                  |{"time":"","stream":"stderr","log":"$LOG_ACTIVATION_SENTINEL\n"}""".stripMargin('|')
+                  |{"time":"","stream":"stdout","log":"$ActionLogDriver.LOG_ACTIVATION_SENTINEL\n"}
+                  |{"time":"","stream":"stderr","log":"$ActionLogDriver.LOG_ACTIVATION_SENTINEL\n"}""".stripMargin('|')
```

Comment:
The `ActionLogDriver` unit test isn't wrong - but doesn't fully cover the area of sentinel processing. Our goal was to keep the `ActionLogDriver` and the existing unit tests as is. So we aligned our new log processing code around `ActionLogDriver` - but externalized the `LOG_ACTIVATION_SENTINEL` such that we can use it in our new unit tests for `DockerClientWithFileAccess`.

In summary, no urgent need for changes in `ActionLogDriver` and corresponding unit tests.